app/src/main/java/com/asfoundation/wallet/verification/ui/paypal/VerificationPaypalActivity.kt

```diff
@@ -14,7 +14,8 @@ class VerificationPaypalActivity : BaseActivity() {
     super.onCreate(savedInstanceState)
     setContentView(R.layout.activity_wallet_paypal_verification)
 
-    setTitle(R.string.verification_settings_unverified_title)
+//    setTitle(R.string.verification_settings_unverified_title)
+    setTitle("")
```

Comment:
why keep it?

Reply:
removed 👍 they were redundant with the label on the manifest